lib/tools/createArtist.tsx

```diff
@@ -0,0 +1,38 @@
+import { z } from "zod";
+import { tool } from "ai";
+import upsertArtist from "./upsertArtist";
+import { ArtistToolResponse } from "@/types/Tool";
+
+const createArtist = (question: string, email: string) =>
+  tool({
+    description: `IMPORTANT: Always call this tool for ANY question related to creating artist.:
+    Do NOT attempt to answer questions on these topics without calling this tool first.
+
+    Example questions that MUST trigger this tool:
+    - "Create a new aritst"
+    - "I wanna create a new artist."
+
+    When in doubt, call this tool to ensure you have the most up-to-date and accurate information.`,
```

Comment:
We want this function to fire in very specific contexts right?
- actual: `when in doubt` (broad trigger)
- expected: specific trigger
 
```suggestion
`,
```